File: 151001/Manchik/publisher/src/main/java/application/dto/StoryResponseTo.java
package application.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.sql.Timestamp;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class StoryResponseTo implements Serializable {
    private Long id;
    private String title;
    private String content;
    private Timestamp created;
    private Timestamp modified;
    private Long authorId;
    private Long markerId;
}

File: 151004/Ermolovich/DC_Lab3/publisher/DC_Lab3/DC_Lab1/DC_Lab1/DTO/StoryResponseTo.cs
using DC_Lab1.DTO.Interface;
using DC_Lab1.Models;

namespace DC_Lab1.DTO
{
    public class StoryResponseTo(int Id, int? authorId, string? Title, string? Content, string? Created, string? Modified ,ICollection<Post> Messages, Author? Author) : IResponseTo
    {
        public int Id { get; set; } = Id;

        public int? authorId { get; set; } = authorId;

        public string? Title { get; set; } = Title;

        public string? Content { get; set; } = Content;

        public string? Created { get; set; } = Created;

        public string? Modified { get; set; } = Modified;

        public virtual ICollection<Post> Messages { get; set; } = Messages;

        public virtual Author? Author { get; set; } = Author;
    }
}
